### ISSUE TITLE
No field access completion support for anonymous function call expression 

### ISSUE THREAD

**Issue by malinthar** [ballerina-platform/ballerina-lang#34513]
**Description:**
![anon](https://user-images.githubusercontent.com/35211477/148169892-f19589ee-be9f-439e-bf95-7fb954e06ad6.png)

**Steps to reproduce:**
```Ballerina
function testAnon() {
    function (int) returns string myFunc = function(int arg)  returns string {return "abc";};
    string result  = myFunc().<cursor>
}
```
**Affected Versions:**
stage-sl-snapshot

**Comment by IMS94** [comment]
I think this will be fixed with #34505.

**Comment by malinthar** [comment]
> I think this will be fixed with #34505.

Checked the changes locally. Seems it is not fixed.
